fbw-a380x/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -63,6 +64,14 @@ export class FMA extends DisplayComponent<{ bus: EventBus; isAttExcessive: Subsc
 
   private trkFpaDeselected = Subject.create(false);
 
+  private baroCorrectedAltitude = Subject.create(0);
```

Comment:
Can you name it just altitude please? After https://github.com/flybywiresim/aircraft/pull/9659 it will be the PFD displayed altitude from the selected source :).